include/deal.II/grid/grid_tools.h

```diff
@@ -1641,6 +1642,43 @@ namespace GridTools
     const std::vector<Point<spacedim>> &                   points);
 
 
+  /**
+   * Given a covering rtree (see GridTools::Cache::get_covering_rtree()), and an
+   * array of points, for each point guesses which processes might own it.
```

Comment:
```suggestion
   * array of points, find a superset of processes which, individually, may own the cell containing the point.
```
We should be more precise than 'guess'. The description I gave is correct, right?